### ISSUE TITLE
Elsa 2 - Workflow Instances checkbox

### ISSUE THREAD

**Issue by riatdyl** [elsa-workflows/elsa-core#998]
Hello,

In the Workflow Instances tab, there's a weird behavior of the checkbox that select every instance.

This happens if there's more than one page of instances.

If I check the box, everything is selected : 
![image](https://user-images.githubusercontent.com/43779018/118477876-804f3880-b70f-11eb-99a0-7081422b827f.png)

And if I uncheck one of the boxes, the first one is disabled : 
![image](https://user-images.githubusercontent.com/43779018/118477971-9eb53400-b70f-11eb-8540-5082c30e6a8f.png)

At this point, there's no problem.

But if I check the first box to select everything, then switch on the second page, this happens : 
![image](https://user-images.githubusercontent.com/43779018/118478092-c73d2e00-b70f-11eb-9693-6c1a925b08a7.png)

This mean that we can't delete multiple pages of instances at once.

**Comment by sfmskywalker** [comment]
Indeed. We can improve this with support for selecting records across multiple pages.
